app/Controllers/statsController.php

```diff
@@ -67,18 +67,25 @@ public function indexAction(): void {
 
 		$feedByCategory = [];
 		$feedByCategory_calculated = $statsDAO->calculateFeedByCategory();
-		for ($i = 0; $i < count($feedByCategory_calculated); $i++) {
+
+		$iMax = count($feedByCategory_calculated);
+		for ($i = 0; $i < $iMax; $i++) {
```

Comment:
This does not correspond to the topic of the PR on improving PHPDoc (which was already quite vague).
I am not so found of PRs with various random changes, and prefer focussed PRs doing one thing well